client/src/components/Header.jsx

```diff
@@ -1,18 +1,20 @@
 import Button from "./Button";
-import PropTypes from 'prop-types';
-const Header=()=>{
-    return(
-        <header className="flex bg-blue-500 text-white p-4 justify-between">
+const Header = () => {
+  return (
+    <header className="flex bg-blue-500 text-white p-4 justify-between">
+      <Button to="/">
         <h1 className="content-start text-2xl font-bold">Feedback Page</h1>
-        <dev className="right-side">
-        <Button className="content-end" to='/create-feedback'>Create Feadback</Button>
-        <Button className="content-end" to='sign'>LogIn</Button>
```

Comment:
- old <Button className="content-end" to='sign'>LogIn</Button>
+ new <Button className="content-end" to='/sign'>LogIn</Button>

Reply:
just changed the path

Reply:
should I change it or did you chang it?